opengever/maintenance/scripts/bumblebee_installation.py

```diff
@@ -139,6 +144,11 @@ def main(app, argv=sys.argv[1:]):
                 annotations[DOCUMENT_CHECKSUM_ANNOTATION_KEY] = version_checksum
                 archived_obj._p_changed = True
 
+            if options.intermediate_commit:
+                if index % options.intermediate_commit == 0:
```

Comment:
Just FYI, this will also commit at 0 (zero) items, at the very start of the loop. Usually that's not an issue, but if that's undesired behavior the condition would need to be tweaked.